docs/dev/agent_omnibus.md

```diff
@@ -80,66 +80,31 @@ outside the Agent repo. By default Omnibus stores packages in the project folder
 itself: running the task multiple times would recursively add those artifacts to
 the source files for the `datadog-agent` software definition.
 
-## Building on Windows
+## Building inside Windows Docker image (Windows host only)
```

Comment:
```suggestion
## Building inside Windows Docker image (Windows host only, recommended)
```

Reply:
There's no other documented way to build on Windows, I don't think we need to say it's "recommended"

Reply:
Fixed in latest commit